sympy/physics/mechanics/tests/test_joint.py

```diff
@@ -0,0 +1,232 @@
+from sympy import sin, cos, Matrix, sqrt, pi, expand_mul, Abs, acos, S
+from sympy.core.symbol import symbols
+from sympy.physics.mechanics import dynamicsymbols, Body, PinJoint
+from sympy.physics.mechanics.joint import Joint
+from sympy.physics.vector import Vector, ReferenceFrame
+from sympy.testing.pytest import raises
+
+t = dynamicsymbols._t
+
+
+def test_Joint():
+    parent = Body('parent')
+    child = Body('child')
+    raises(TypeError, lambda: Joint('J', parent, child))
```

Comment:
Why would this raise a type error?

Reply:
Making an instance of abstract class gives TypeError by python itself

Reply:
ok